src/main/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminAuditController.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.dts.fact.controllers.admin;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.dts.fact.config.security.Role;
+import uk.gov.hmcts.dts.fact.model.admin.Audit;
+import uk.gov.hmcts.dts.fact.services.admin.AdminAuditService;
+
+import java.util.List;
+
+import static org.springframework.http.ResponseEntity.ok;
+import static uk.gov.hmcts.dts.fact.services.admin.AdminRole.FACT_ADMIN;
+import static uk.gov.hmcts.dts.fact.services.admin.AdminRole.FACT_SUPER_ADMIN;
+
+@RestController
+@RequestMapping(
+    path = "/admin/audit",
+    produces = {MediaType.APPLICATION_JSON_VALUE}
+)
+public class AdminAuditController {
+    private final AdminAuditService adminAuditService;
+
+    @Autowired
+    public AdminAuditController(AdminAuditService adminAuditService) {
+        this.adminAuditService = adminAuditService;
+    }
+
+    @GetMapping()
+    @ApiOperation("Find All Audits")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Successful", response = Audit.class, responseContainer = "List"),
+        @ApiResponse(code = 401, message = "Unauthorized"),
+        @ApiResponse(code = 403, message = "Forbidden")
+    })
+    @Role({FACT_ADMIN, FACT_SUPER_ADMIN})
+    public ResponseEntity<List<Audit>> getAudits() {
+        return ok(adminAuditService.getAllAuditData());
```

Comment:
Should we paginate the data in the controller? Over time, I suspect the audit table will have a huge number of rows, even if we purge data over n months old.

Reply:
Also, should admin (non-super) have access to this data?